app/presenters/result_set_presenter.rb

```diff
@@ -78,7 +85,7 @@ def best_bet?
     end
   end
 
-  def sort_option
+  def unpresented_sort_option
```

Comment:
I renamed this so it would be really clear what it is, even though it makes the overall bigger.